washee_app/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -81,7 +81,7 @@
       buildConfiguration = "Debug">
    </AnalyzeAction>
    <ArchiveAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
Vil denne her ændring ikke få os til ikke at være IPhone compatible?

Hvis ideen er at det skal ændres alt efter "production" eller "development", så lyder det som noget man kan putte i en environment fil (ved dog ikke hvordan man gør ;) )

Reply:
Tænker specifikt at flutter må have en eller anden form for environment control. Så kan man derefter evt. opdatere up.bat og up.command scriptsene til at de starter op i "dev mode" og at man laver et "production build" script der bruger production environment variablene.

Dette tænker jeg at oprette som en PR task, der evt. kan ses på EFTER forsøget d. 18. april er sat i gang (det er udenfor scope af det her sprint)

Reply:
Skiftet sker når man uploader til app store, den Pia fil som genereres af Xcode til appstore kan KUN være i release, og det sættes automatisk.
den pågældende setting gør at jeg under debug også ser det samme som uploades.